.github/workflows/kustomize-lint.yaml

```diff
@@ -32,7 +33,7 @@ jobs:
           cd /home/runner/work/vse-carslab-hub/vse-carslab-hub/cnf-features-deploy/ztp/policygenerator-kustomize-plugin
           make build KUSTOMIZE_DIR=/home/runner/kustomize POLICYGEN_KUSTOMIZE_DIR=/home/runner/kustomize/plugin/ran.openshift.io/v1/policygentemplate
       - name: Code Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
```

Comment:
Should we be concerned that this line is v3 in the 2nd main (cars2) branch?

Reply:
Not urgent, but we should update it there as well

Reply:
will tackle next time I do a PR there.